packages/tldraw/src/lib/ui/components/primitives/TldrawUiInput.tsx

```diff
@@ -108,6 +110,7 @@ export const TldrawUiInput = React.forwardRef<HTMLInputElement, TLUiInputProps>(
 			(e: React.KeyboardEvent<HTMLInputElement>) => {
 				switch (e.key) {
 					case 'Enter': {
+						if (isCompositioning.current) return
```

Comment:
Since this is a specific fix for Chrome, let's add a comment here specifying why this is necessary. It'd be great to mention that it's a Chrome issue, specifically tied with the `e.currentTarget.blur()` below that causes the double-text.